lib/ui/molecules/message/list_item.dart

```diff
@@ -36,6 +37,10 @@ class MessageListItem extends StatelessWidget {
                 ),
               ],
             ),
+          if (!message.isMe)
+            ProfileImage(
+              image: null,size: 40,
```

Comment:
nits 細かいですが
```suggestion
              image: null, size: 40,
```
因みにdartの自動整形かかってますか...？